Correcting some HP to HPE logo

FTBZ opened the pull request:
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

- Edited some HP device to use the HPE logo without changing any text that can break other things.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/d4e91fe0-fb62-4b34-999c-0316cfcdbf59) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/d4e91fe0-fb62-4b34-999c-0316cfcdbf59) completed: **No new issues**

laf commented:
imho this logo isn't as good as the old hp.svg one :(

FTBZ commented:
Yes, but it's no longer the same company...I used the logo from @murrant in the PR #5400. 

I noticed that HPE is using only the green rectangle in some infrastructure. Perhaps it's better?

murrant commented:
HPE has the option to have just the green rectangle, should we use that?

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/2b5806a6-a93d-4039-b2be-aa5c14e2771f) was created.

FTBZ commented:
I changed the logo to only the rectangle. It's not correct with the Terms of use, but this is the case for the majority of logos and HPE itself use this in infrastructure interface like vCenter.

I just added two devices on my dev VM to show it:

![screen shot 2017-01-24 at 15 34 12](https://cloud.githubusercontent.com/assets/1927109/22251417/f77a21de-e24a-11e6-9ef3-31322e34f798.png)

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/2b5806a6-a93d-4039-b2be-aa5c14e2771f) completed: **No new issues**

murrant commented:
@FTBZ Perhaps #5573 might interest you :)

laf commented:
My personal opinion is we stick with the current hp logo but I'm not a user of any hpe stuff in LibreNMS so take my answer loosely

murrant commented:
@laf actually, we should be using both as they are two separate things now.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/f2a8f654-1032-4204-a333-fc6c8b2c2e66) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/f2a8f654-1032-4204-a333-fc6c8b2c2e66) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/36c08975-fcad-4945-a14a-b1efd641eed4) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/36c08975-fcad-4945-a14a-b1efd641eed4) completed: **No new issues**

laf commented:
Well yes because of your PR but that wasn't the case before.

murrant commented:
@laf I meant both HP and HPE logos.  HP is basically printers, HPE is enterprise stuff.

laf commented:
I know the companies are two separate things but that HPE logo was pretty poor and we have a choice on using one or the other.

murrant commented on html/images/os/hpe.svg:
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="300" height="300" viewBox="0 0 300 300"><path fill="#00B188" d="M277.3 173.6H22.7v-47.2h254.6v47.2zm18.2-65.4H4.5v83.6h291v-83.6z"/></svg>
```
I wonder if this should have all the padding removed...

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/8f307b09-5fd0-4b26-af25-92e69bb16f8c) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/librenms/librenms/inspections/8f307b09-5fd0-4b26-af25-92e69bb16f8c) completed: **No new issues**
